### DISCUSSION TITLE
Use container/vms notes for the documentation, especially Web UI links

### DISCUSSION THREAD

**Discussion by mohnen** [tteck/Proxmox#346] [Ideas]
My idea is to extend the scripts such that they create Notes for the container/vm.
I did this manually, see picture

<img width="846" alt="2022-07-15 15_54_31-pve - Proxmox Virtual Environment" src="https://user-images.githubusercontent.com/4920720/179237499-2e77cdd8-c797-48e9-9ba6-9907d83df962.png">

**Comment by tteck** [comment]
That's a great idea, and I can do that in the scripts. It's just a lot of scripts to go back through and redo.

**Reply by mohnen** [reply]
Btw, great stuff you did!
Do you want me to make an issue it of the idea?

**Reply by tteck** [reply]
Idea is enough for now. I'll do the Adguard script and post it here for you to test, then approve or disapprove.

**Reply by tteck** [reply]
Can you paste those notes here? I'll just use what you already have

**Reply by mohnen** [reply]
Here u go:

# AdGuard Home LXC

AdGuard Home Setup Interface: [http://192.168.178.5:3000](http://192.168.178.5:3000)

AdGuard Home User Interface: [http://192.168.178.5](http://192.168.178.5)

(For the Home Assistant Integration, use port 80 not 3000)

⚙️ To Update Adguard

Update from the Adguard UI

**Reply by mohnen** [reply]
```
# AdGuard Home LXC

AdGuard Home Setup Interface: [http://192.168.178.5:3000](http://192.168.178.5:3000)

AdGuard Home User Interface: [http://192.168.178.5](http://192.168.178.5)

(For the Home Assistant Integration, use port 80 not 3000)

⚙️ To Update Adguard

Update from the Adguard UI

```

**Reply by tteck** [reply]
Try this
```

```

**Comment by mohnen** [comment]
It works perfectly for me!

![2022-07-15 16_48_57-pve - Proxmox Virtual Environment](https://user-images.githubusercontent.com/4920720/179248305-3e81ea6e-f657-47cc-b300-81385902b6e3.png)

**Reply by tteck** [reply]
Good! I'll start adding notes.
Thanks for the great idea!

**Reply by tteck** [reply]
Don't think I'll add "To Update". An update script may change paths

**Reply by mohnen** [reply]
Hm, good point. Maybe you can update the notes? But this  would mean detecting the part which was written by the script in case the user added more notes

**Reply by tteck** [reply]
Another thing, if you're using DHCP and the IP changes, those notes are useless. May cause more headaches than it's worth.
(Just thinking about troubleshooting rookie users)

**Reply by mohnen** [reply]
Sounds very reasonable. 

I have created a script, which creates a synopsis of all lxcs in the node's notes.
Maybe you want to check it out: https://github.com/mohnen/Proxmox/blob/main/pve_synopsis

The result looks like this on my PVE
![image](https://user-images.githubusercontent.com/4920720/181362241-d3e812ad-181f-4b16-8720-5e6847f7dd58.png)

**Reply by mohnen** [reply]
I have update the way the information is presented in the synopsis

<img width="738" alt="image" src="https://user-images.githubusercontent.com/4920720/181478741-8fd4026f-da08-4313-ac25-2b82e326882c.png">
